veejay-current/veejay-server/man/veejay.1

```diff
@@ -567,22 +558,45 @@ Stop recording
 .TP
 .B [CTRL] + [f]
 Toggle fullscreen / windowed mode
+.SH EXAMPLES
+.TP
+.B veejay -u |less
+Startup veejay and list all events (VIMS/OSC) and effect descriptions
+.TP
+.B veejay -p 4000 ~/my_video1.avi
+Startup veejay listening on port 4000 (use this to use multiple veejays)
+.TP
+.B veejay -d -W 352 -H 288 -R 25 -N 0
+Startup veejay using dummy video at 25 frames per second, dimensions 352x288
+and using PAL.
+.TP
+.B veejay movie1.avi -V 224.0.0.50 -p 5000 -n -v
+Startup veejay, using multicast protocol on port 5000, with autolooping
+and no colored verbose output
+.TP
+.B veejay --output 5 --output-file /dev/video1
+Startup veejay headless and write video output to a vloopback device
+.TP
+.B veejay --output 4 --output-file /tmp/video-yuv4mpeg.pipe
+Startup veejay headless and write yuv4mpeg video to a named fifo pipe
```

Comment:
did you test if these examples actually work ? if not, we should create another issue and test veejay by documentation

Reply:
#230